node/core/pvf/src/execute/queue.rs

```diff
@@ -149,7 +149,7 @@ impl Queue {
 						break;
 					}
 				}
-				ev = self.mux.select_next_some() => handle_mux(&mut self, ev).await,
+				ev = self.mux.select_next_some() => handle_mux(&mut self, ev),
```

Comment:
Please avoid adding too many unrelated changes as it worsens review experiense as well as commit history after squash.